src/mtcp/restore_libc.c

```diff
@@ -503,6 +551,7 @@ TLSInfo_VerifyPidTid(pid_t pid, pid_t tid)
 {
   pid_t tls_pid, tls_tid;
   char *addr = (char *)get_tls_base_addr();
+  addr = (char *)get_tls_base_addr();
```

Comment:
Why are we doing this again? Isn't the line above (line no. 553) doing the same assignment?

Reply:
Thanks.  This must have been a typo.